CMakeLists.txt

```diff
@@ -0,0 +1,54 @@
+cmake_minimum_required(VERSION 2.8)
+
+cmake_policy(SET CMP0048 NEW)
+
+project(safeside VERSION 0.1.0 LANGUAGES CXX)
+
+
+add_library(safeside demos/cache_sidechannel.cc demos/instr.cc)
+
+# Spectre V1 -- bounds check bypass
+add_executable(spectre_v1 demos/spectre_v1.cc)
+
+# Spectre V3 / Meltdown
+add_executable(meltdown demos/meltdown.cc)
+
+# Spectre V4 -- speculative store bypass
+add_executable(spectre_v4 demos/spectre_v4.cc)
+
+
+target_link_libraries(spectre_v1 safeside)
```

Comment:
For my education, is this normal with CMake, or is it more expected to put the target_link_libraries next to the add_executable that it's for? I have a preference for moving them next to the individual add_executable, since that's what I'm used to from e.g. Bazel's BUILD files. (And it also seems to me that it makes it easier to update when one adds a new library dependency.)

Reply:
I have seen it done both ways, although from a maintenance point of view you are correct. I can make those changes.

Reply:
done.